samples/templates/default-azuredeploy-docker.json

```diff
@@ -27,20 +27,14 @@
         "appServicePlanSku": {
             "type": "string",
             "allowedValues": [
-                "F1",
-                "D1",
-                "B1",
-                "B2",
-                "B3",
```

Comment:
B3 is currently the value used in provision-deploy.yml. That file needs to be updated.

Reply:
I added B3 back to the list. PR build and deploy pipeline successfully completed.